src/components/Gesture.svelte

```diff
@@ -270,45 +283,77 @@
           <CloseIcon class="text-xl m-1" />
         </IconButton>
       </div>
-      <h3
-        class="px-2"
-        contenteditable
-        bind:innerText={$nameBind}
-        on:click={titleClicked}
-        on:keypress={onTitleKeypress} />
-    </div>
+    {/if}
+    <label for="gestureName" class="sr-only"
+      >{$t('content.data.addAction.inputLabel')}</label>
+    <input
+      use:init
+      name="gestureName"
+      class="w-40 col-start-2 p-2 col-end-5 transition ease rounded bg-gray-100 placeholder-gray-500 outline-primary"
+      contenteditable
+      id="gestureName"
+      placeholder={gesturePlaceholderName}
+      bind:value={$nameBind}
+      on:keypress={onTitleKeypress} />
   </div>
 </GestureTilePart>
 
-<div class="max-w-max">
-  <GestureTilePart small elevated>
-    <div class="h-full flex items-center gap-x-3 p-2">
-      <div class="w-33 flex justify-center items-center gap-x-3">
-        <IconButton
-          ariaLabel={$t(
-            $chosenGesture === gesture
-              ? 'content.data.recordAction'
-              : 'content.data.selectAction',
-            {
-              values: {
-                action: $nameBind,
+{#if showAddActionWalkThrough}
+  <div
+    class="h-full flex w-50 flex-col relative items-center"
+    style="transform: translate(-50px, 50px)">
+    <img
+      class="mb-3 w-30"
+      alt={$t('content.data.addActionWalkThrough.imageAlt')}
```

Comment:
I think we can have empty alt text here `alt=""` as the emoji is purely decorative and there is descriptive text below.

Reply:
Makes sense, I was struggling to come up with alts because of the repetition with the text below https://github.com/microbit-foundation/ml-trainer/pull/103/commits/6f155e8cca85f2a96f5b76909a8008eb40747645